back/node_watcher/src/nodeWatcher.ts

```diff
@@ -147,5 +143,5 @@ export async function nodeWatcher(tasks: NomidotTask[]): Promise<void> {
   const provider = new WsProvider(ARCHIVE_NODE_ENDPOINT);
   const api = await ApiPromise.create({ provider });
 
-  incrementor(api, provider, tasks);
+  return incrementor(api, provider, tasks);
```

Comment:
incrementor is the culprit, it didn't have a `.catch`, and we got UnhandledPromiseRejectionWarning.

Now we return this, and it gets handled here: https://github.com/paritytech/Nomidot/blob/master/back/node_watcher/src/index.ts#L13